iceoryx_examples/callbacks/README.md

```diff
@@ -197,21 +197,22 @@ Afterward, we reset both caches to start fresh again.
 ### Additional context data for callbacks (ice_callbacks_listener_as_class_member.cpp)
```

Comment:
How is this file related to the design decision document? Doesn't it violate our traceability rule when we change files which are unrelated to the issue?!

Reply:
Just install a markdown linter and everything will be fine :stuck_out_tongue: 

IMHO trailing white spaces can be removed as a drive-by-refactoring. I can also move this to another issue.

You'll get a free hug :hugs: if you install a markdown linter!

Reply:
I did it already some weeks ago. Can I get a :cookie: instead?

Reply:
@mossmaurice passed @elfenpiff :cookie: and a :cocktail:

Reply:
Great you volunteered to create a PR that fixes all the trailing white-spaces in `.md` :smiley: